src/features/Instructors/ManageInstructorsPage/AssignSection.jsx

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable react/prop-types */
+import React, {
+  useMemo, useState, useEffect, forwardRef, useRef, useImperativeHandle,
+} from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import {
+  Container, Row, Col, DataTable, Pagination, Button,
+} from '@edx/paragon';
+import InstructorsFilters from 'features/Instructors/InstructorsFilters';
+import ControlledSelect from 'features/Instructors/ManageInstructorsPage/ControlledSelect';
+
+import { fetchInstructorsData } from 'features/Instructors/data';
+import {
+  updateCurrentPage,
+} from 'features/Instructors/data/slice';
+
+import { columns } from 'features/Instructors/ManageInstructorsPage/columns';
+import { initialPage, RequestStatus } from 'features/constants';
+
+// Component to get access the clearSelection function
+const ClearAction = ({ tableInstance, clearSelectionRef }) => (
+  <Button
+    style={{ display: 'none' }}
+    onClick={() => {
+      tableInstance.clearSelection();
+    }}
+    ref={clearSelectionRef}
+  >
+    Clear Selection
+  </Button>
+);
+
+const AssignSection = forwardRef((props, ref) => {
+  const dispatch = useDispatch();
+  const clearSelectionRef = useRef(null);
+  const selectedInstitution = useSelector((state) => state.main.selectedInstitution);
+  const stateInstructors = useSelector((state) => state.instructors.table);
+  const [currentPage, setCurrentPage] = useState(initialPage);
+  const COLUMNS = useMemo(() => columns, []);
+  const isLoading = stateInstructors.status === RequestStatus.LOADING;
+
+  const selectColumn = {
+    id: 'selection',
+    Header: <></>, // eslint-disable-line react/jsx-no-useless-fragment
+    Cell: ControlledSelect,
+    disableSortBy: true,
+  };
+
+  const handlePagination = (targetPage) => {
+    setCurrentPage(targetPage);
+    dispatch(updateCurrentPage(targetPage));
+  };
+
+  const resetPagination = () => {
+    setCurrentPage(initialPage);
+  };
+
+  useImperativeHandle(ref, () => ({
+    clearSelectionFunc: () => clearSelectionRef?.current?.click(),
+  }));
+
+  useEffect(() => {
+    if (Object.keys(selectedInstitution).length > 0) {
+      const instructorFilters = stateInstructors.filters;
+      dispatch(fetchInstructorsData(selectedInstitution.id, currentPage, instructorFilters));
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [currentPage, selectedInstitution, dispatch]);
+
+  return (
+    <Container size="xl" className="p-4 mt-3 instructors-content">
+      <h3>Assign / Add instructors</h3>
+      <InstructorsFilters isAssignSection resetPagination={resetPagination} />
```

Comment:
There's some design issues with this component:
- Here should have a magnifying glass icon 
![missing icon in input](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/52179095/78c5ef6c-3e05-4e6b-b76a-ea83ac5a702e)

- The buttons should be next to input and not behind it

![buttons in wrong posittion](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/52179095/921cac9a-ee06-41d7-aab5-08b15e2a1e88)